.github/workflows/ci.yml

```
@@ -10,41 +10,16 @@ on:
 jobs:
   run:
-    env:
-      views_path: /cvmfs/sft.cern.ch/lcg/views
-      platform: x86_64-el9-gcc13-opt
     runs-on: ubuntu-24.04
-    strategy:
-      matrix:
-        lcg: [LCG_106]
+    container:
+      image: gitlab-registry.cern.ch/linuxsupport/alma9-base
+      volumes: /cvmfs:/cvmfs:shared
```

actionlint (1.7.4)

"volumes" section must be sequence node but got scalar node with "!!str" tag

(syntax-check)